master/internal/user/postgres_users_intg_test.go

```diff
@@ -0,0 +1,323 @@
+//go:build integration
+// +build integration
+
+package user
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/etc"
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+var ctx = context.TODO()
```

Comment:
I would just define this in the scope it is needed, otherwise refactoring stuff to actually use it / test with a ctx will be really awkward.